File: setup/psql/add_password.py
import bcrypt
import psycopg2

CONNECTION_STRING = "dbname=minera_catalog user=postgres"

def main():
    password = b"asd"
    hashed_password = bcrypt.hashpw(password, bcrypt.gensalt()).decode("UTF-8")

    try:
        connection = psycopg2.connect(CONNECTION_STRING)
        cursor = connection.cursor()
    except:
        print("connection failure")
        return

    query_string = "INSERT INTO login (password) VALUES (%s);"
    query_params = (hashed_password,)

    try:
        cursor.execute(query_string, query_params)
        connection.commit()
    except:
        print("execution error")
        return

    print("ok")

main()

File: setup/psql/AddPassword.go
package main

import (
	"fmt"
	"database/sql"
	_ "github.com/lib/pq"
	"golang.org/x/crypto/bcrypt"
)

var connStr string = "user=postgres dbname=minera_catalog host=/run/postgresql"

func main() {
	password := []byte("asd")

	// generate hash
	hash, err := bcrypt.GenerateFromPassword(password, bcrypt.DefaultCost)
    if err != nil { fmt.Println(err); return }

	// connect to database
	db, err := sql.Open("postgres", connStr)
    if err != nil { fmt.Println(err); return }
	defer db.Close()

	// insert password into database
	_, err = db.Exec(`INSERT INTO login (password) VALUES ($1)`, string(hash)) 
    if err != nil { fmt.Println(err); return }

	fmt.Println("ok")
}
